delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/ProjectsImpl.scala

```diff
@@ -33,15 +34,20 @@ final class ProjectsImpl private (
   override def create(
       ref: ProjectRef,
       fields: ProjectFields
-  )(implicit caller: Subject): IO[ProjectResource] = {
+  )(implicit caller: Subject): IO[ProjectResource] =
     for {
       resource <- eval(CreateProject(ref, fields, caller)).span("createProject")
       _        <- scopeInitializations
-                    .parUnorderedTraverse(_.onProjectCreation(resource.value, caller))
+                    .parUnorderedTraverse { init =>
```

Comment:
A bit like `IndexingAction`, it could be interesting to refactor this by introducing some kind of aggregate there which would take care of the init steps and saving errors